txscript/standard_test.go

```diff
@@ -568,26 +573,48 @@ func TestPayToAddrScript(t *testing.T) {
 	}
 
 	//  mainnet p2pk 13CG6SJ3yHUXo4Cr2RY4THLLJrNFuG3gUg
-	p2pkCompressedMain, err := dcrutil.NewAddressSecpPubKey(decodeHex("02192d74"+
```

Comment:
Please don't change this.  There is no need since they are specifically being given the compressed bytes.